stripecardscan/src/main/java/com/stripe/android/stripecardscan/scanui/SimpleScanStateful.kt

```diff
@@ -0,0 +1,51 @@
+package com.stripe.android.stripecardscan.scanui
+
+import android.util.Log
+import com.stripe.android.camera.framework.AnalyzerLoopErrorListener
+import com.stripe.android.stripecardscan.framework.Config
+
+internal interface SimpleScanStateful {
+
+    var scanStatePrevious: ScanState?
+    var scanState: ScanState
+    val scanErrorListener: ScanErrorListener
+
+    /**
+     * The state of the scan flow. This can be expanded if [displayState] is overridden to handle
+     * the added states.
+     */
+    abstract class ScanState(val isFinal: Boolean) {
+        object NotFound : ScanState(isFinal = false)
+        object FoundShort : ScanState(isFinal = false)
```

Comment:
I don't think we actually use `FoundShort` anywhere. We can probably just remove this.